src/Fantomas/Context.fs

```diff
@@ -27,6 +27,10 @@ type ShortExpressionInfo =
         - x.StartColumn > x.MaxWidth // expression is not too long according to MaxWidth
         || (currentColumn > maxPageWidth) // expression at current position is not going over the page width
 
+type Size =
+    | CharacterWidth of width: Num * maxWidth: Num
```

Comment:
Why does this have two values?

Reply:
It was for uniformity with the other case, but I can remove it since it turns out never to be used.

Reply:
Changed.